samples/hosting/docker/sample.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Hosting endpoints in Docker Linux containers
+summary: Hosting multiple endpoints in several Docker Linux containers managed by Docker Compose
+reviewed: 2017-09-25
+component: Core
+tags:
+- Hosting
+related:
+- nservicebus/hosting
+---
+
+This sample demonstrates how to use Docker Linux containers to host NServiceBus endpoints communicating over the RabbitMQ transport.
+
+## Prerequisites
+
+This sample requires that the following tools are installed:
+
+  - [.NET Core 2.0 SDK](https://www.microsoft.com/net/download/core)
+  - [Docker Community Edition](https://www.docker.com/community-edition) or higher
+
+NOTE: Container that runs the RabbitMQ broker is a Linux container and [Docker for Windows also needs to be configured to use Linux containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
+
+## Running the sample
+
+The sample requires .NET Core CLI tools and Docker CLI tools to build and run the code.
```

Comment:
This is another prerequisite, it should be in the prereqs section above. In order to preview the chunks ahead, this line should be something like "Running the sample involves building the binaries, then building the Docker containers, and finally starting the containers."